### DISCUSSION TITLE
Welcome to backtrader_plotting Discussions!

### DISCUSSION THREAD

**Discussion by verybadsoldier** [verybadsoldier/backtrader_plotting#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gahoccode** [comment]
Hi!
How do I pass a df with a mm/dd/yyyy format using this example 
data = bt.feeds.YahooFinanceCSVData(
        dataname="orcl-1995-2014.txt",
        fromdate=datetime.datetime(2000, 1, 1),
        todate=datetime.datetime(2001, 2, 28),
        reverse=False,
        swapcloses=True,
    )

This is my dataframe 
df = investpy.get_stock_historical_data(stock='ree',
                                        country='Vietnam',
                                        from_date='01/03/2015',
                                        to_date='21/03/2022',
                                        interval= 'daily')
